.gitignore

```diff
@@ -22,18 +22,11 @@ dash/include/dash/util/StaticConfig.h
 *.out
 ipm/*/GEN.*
 my.*.sh
-.syntastic*
-.clang*
 tags
 *.orig*
 *.bac*
 *.mine*
 RELEASE/*
-dash/include/dash/util/StaticConfig.h
```

Comment:
If I remember correctly, this file is generated by @fuerlinger's build scripts, so it should be ignored.

Reply:
Yup, but the entry was duplicated :)